versioned/tiered/rocksdb/src/main/java/org/projectnessie/versioned/rocksdb/RocksDBStoreConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.versioned.rocksdb;
+
+import org.immutables.value.Value.Default;
+import org.immutables.value.Value.Immutable;
+
+@Immutable
+public abstract class RocksDBStoreConfig {
+
+  @Default
+  public String getDbDirectory() {
+    return "";
+  }
+
+  public static ImmutableRocksDBStoreConfig.Builder builder() {
```

Comment:
I was looking at Immutables and wonder, do we actually need to have a builder() method defined, or will it generate this for us?

Reply:
Indeed it looks like it Immutable can generate the builder for us. I remove the builder() method from here and in Function.